cmake/externals/projects_modules/VTK.cmake

```diff
@@ -1,76 +0,0 @@
-################################################################################
```

Comment:
I has never been used (except with medInria). The idea was to allow people that cannot install things on their machine to still compile OM (in the same way we provide lapack, matio, hdf5, and zlib). I'm for keeping it and allow its use which should be extremely simple....